src/main/java/dev/dias375/swplanetsapi/SWAPI/model/Climate.java

```diff
@@ -0,0 +1,7 @@
+package dev.dias375.swplanetsapi.SWAPI.model;
+
+public enum Climate {
+
+    Dry, Cold, Tempered
```

Comment:
[NB] Eu te sugeriria implementar um método fromName() aqui. É um padrão simples de enum que evita que vc precise receber exatamente o nome certo de cada item do enum. Além disso, é boa prática que os itens do enum sejam definidos em caixa alta, já que se tratam essencialmente de constantes. 

Aqui é uma bom ponto pra começar a ver sobre o fromName(): https://stackoverflow.com/questions/604424/how-to-get-an-enum-value-from-a-string-value-in-java